src/components/ui/Modal/ModalHeader.tsx

```diff
@@ -35,7 +35,7 @@ export const ModalHeader: FC<ModalHeaderProps> = ({
         {title}
       </Tag>
 
-      {displayClose && (
+      {(displayClose && !context.dismissible) && (
```

Comment:
I think we would want to display a close button in the modal *is* dismissable.

Reply:
Agree, updating.